example/config/configuration.go

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (c) 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package config
+
+import (
+	"errors"
+)
+
+// This file contains example of custom configuration that can be loaded from the service's configuration.toml
+// and/or the Configuration Provider, aka Consul (if enabled).
+// For more details see https://docs.edgexfoundry.org/2.0/microservices/device/Ch-DeviceServices/#custom-configuration
```

Comment:
Is there an existing PR to update the docs and provide the specified page anchor (`#custom-configuration`)?

Reply:
Not yet, but have submitted an issue.